examples/custom-types/sailor.php

```diff
@@ -42,6 +43,15 @@ public function makeClient(): Client
                     'withCustomEnum' => CustomEnum::B,
                 ],
             ]);
+            $mockClient->responseMocks[] = static fn (string $query, ?\stdClass $variables): Response => Response::fromStdClass((object) [
```

Comment:
It appears that `$mockClient` is instantiated every multiple times, thus the consecutive tests across different operations in `examples/custom-types/src/test.php` do not work correctly.